sdk/lib/freestanding/memset.c

```diff
@@ -88,3 +88,9 @@ void *__cheri_libcall memset(void *dst0, int c0, size_t length)
 		} while (--t != 0);
 	return (dst0);
 }
+
+void __cheri_libcall explicit_bzero(void *s, size_t n)
+{
+	memset(s, 0, n);
```

Comment:
Does this result in an extra call/return within the library?

Reply:
It should be a tail call. I don’t think we can merge the entries, but the compiler will lower calls within the same compilation unit as direct. I think it may not do tail calls right now, but that’s a simple fix soon.